data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/beacon/GetDepositSnapshot.java

```diff
@@ -41,12 +42,16 @@
 import tech.pegasys.teku.infrastructure.ssz.SszData;
 import tech.pegasys.teku.validator.coordinator.Eth1DataProvider;
 
+/**
+ * Get Deposit Snapshot Tree, see <a
+ * href="https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4881.md">EIP-4881</a>
+ */
 public class GetDepositSnapshot extends MigratingEndpointAdapter {
 
   public static final String ROUTE = "/teku/v1/beacon/deposit_snapshot";
 
   private static final SerializableTypeDefinition<DepositTreeSnapshot> DEPOSIT_SNAPSHOT_TYPE =
-      DepositTreeSnapshot.getJsonTypeDefinition();
+      DepositTreeSnapshot.getJsonTypeDefinition(DEPOSIT_TREE_SNAPSHOT_SCHEMA);
```

Comment:
It seems like the `DepositTreeSnapshot` and `DepositTreeSnapshotSchema` should be in the same module and so we shouldn't need to pass this in here.  The `DepositTreeSnapshot.getJsonDefinition()` is the API I think we want to stick with.